Core/Graphics/Source/GraphicsImpl.h

```diff
@@ -129,6 +133,7 @@ namespace Babylon
         bgfx::Encoder* GetEncoderForThread();
         void EndEncoders();
         void CaptureCallback(const BgfxCallback::CaptureData&);
+        void YFlip();
```

Comment:
I don't think the Graphics component should have this feature/responsibility, at least not long-term. It is `NativeEngine`, not `Graphics`, which wants to render things upside-down, so tethering core behaviors of the `Graphics` backbuffer to `NativeEngine` logic seems like it might be breaking separation of concerns; and, more importantly, I think it might disrupt certain scenarios.

Suppose someone was writing some sort of CAD or other editor-like thing using Babylon Native for 3D rendering and a native UI system for the interface. Babylon Native would render to a portion of the backbuffer and the UI would render to the rest, both in their own separate threads coordinated by the Graphics component. In this situation, I don't think it makes sense for the UI to have to worry about the backbuffer getting flipped; it should just be able to render normally without consideration for what `NativeEngine` may or may not be doing.

Even if my thinking is correct on this, I don't know of any currently-forecast use cases where this would be an issue, so it's not the end of the world if we break this in the short term. If we do that, though, and this does become important, then we'll probably end up having to walk this back eventually.